lib/public/views/Runs/RunPerDataPass/RunsPerDataPassOverviewModel.js

```diff
@@ -143,7 +143,9 @@ export class RunsPerDataPassOverviewModel extends FixedPdpBeamTypeRunsOverviewMo
                     this._skimmableRuns$.setCurrent(RemoteData.failure(error));
                 }
             },
-            Other: () => null,
+            Other: () => {
+                throw new Error('Need to successfully fetch skimmable runs in order to update them');
+            },
```

Comment:
This will simply break the application. You should set a remote data to failure instead. Why not using `apply`?

Reply:
not really, but applied